spec/node/asciidoctor.spec.js

```diff
@@ -397,6 +397,21 @@ Content 2`;
       expect(result).toContain('we have liftoff.');
     });
 
+    it('should be able to create an image block from a processor extension', () => {
+      const registry = asciidoctor.Extensions.create(function () {
+        this.blockMacro(function () {
+          this.named('img');
+          this.process((parent, target) => {
+            return this.createImageBlock(parent, { target: target + '.png' });
+          });
+        });
+      });
+      const opts = {};
+      opts[asciidoctorVersionGreaterThan('1.5.5') ? 'extension_registry' : 'extensions_registry'] = registry;
```

Comment:
`master` should only be built against Asciidoctor core 1.5.6+ and `master`. So we should remove this condition and use `extensions_registry`.

But I think it's better to remove this condition globally (in a dedicated pull request)